tests/typecheck/test_test.yml

```diff
@@ -0,0 +1,9 @@
+- case: test_api_test_case
+  main: |
+    from rest_framework import test, status
+
+    class MyTest(test.APITestCase):
```

Comment:
Can you please check other types as well? You can use https://github.com/typeddjango/pytest-mypy-plugins#2-parametrized

Reply:
Done!

Reply:
Thanks for the link, btw. It was pretty helpful.